packages/server/lib/modes/record.js

```diff
@@ -309,6 +310,16 @@ const createRun = Promise.method((options = {}) => {
   debug('commit information from Git or from environment variables')
   debug(commit)
 
+  const ci = {
+    params: ciProvider.ciParams(),
+    provider: ciProvider.provider(),
+  }
+
+  // write git commit and CI provider information
+  // in its own log source to expose separately
+  logCommitInfo('commit information %o', commit)
```

Comment:
`commit` is already logged line 311, should prob pick one of the other

Reply:
so I wanted this message to be in both - if you are debugging the recording, you probably want to see the commit info too

Reply:
if we are doing DEBUG=cypress:* for debugging purposes, we don't want to see the same info logged out twice

> if you are debugging the recording, you probably want to see the commit info too

now that the namespace has been changed, is this still the case? does `cypress:server:record:ci-info` get printed with `DEBUG=cypress:server:record`?

Reply:
ok, removed duplicate log